### ISSUE TITLE
Request: Bind untiled windows to workspace by default

### ISSUE THREAD

**Issue by runemopar** [rickbutton/workspacer#133]
Love workspacer, makes using Windows a much better experience.

However, workspacer interacts poorly with some applications.

Specifically the entire Jetbrains suite at least on my machine when using IdeaVim is unusable: Whenever I make an edit, the entire window becomes blank for some milliseconds before returning back to normal. This obviously causes a poor experience.

Though this is an issue in and of itself, it could be solved rather elegantly and also allow for untiled windows in general to be usable if the Ctrl+T shorctut did not completely remove the window from being managed by workspacer. I need to do this to work in CLion/IntelliJ, but when I switch workspaces the IDE obscures other windows. So if the window was still bound to the workspace I placed it in by default, this problem would be nonexistent.

TL;DR: Some applications work poorly with workspacer, would be great if I could bind untiled windows to workspaces.

**Comment by rickbutton** [comment]
You might be able to write some clever routing rules to get "untitled windows" to go to a specific workspace.

The issue with JetBrains editors is a known issue:: https://github.com/rickbutton/workspacer/issues/61 .

**Comment by runemopar** [comment]
Sorry, "untiled" (as in not tiled, not untitled) may have been a poor choice of words. I meant that I have to toggle tiling such that the Jetbrains editor is not tiled. However, this means that it will obscure other windows if I switch to a different workspace as it just kind of hangs out wherever you place the window.

The issue you linked seems to not have this problem, so maybe it is a bug?

My suggestion is that you can have those windows that have tiling toggled off still remain in the workspace they were previously assigned so that they automatically minimize when I switch to a different workspace. This would be a nice workaround for jetbrains editors.

In the meantime I guess I can just start work on a tiling engine that has that feature.

**Comment by BeeryShklar** [comment]
It would also be great if we could do something like that:
```c#
context.WindowRouter.AddRoute((window) => 
    window.Title.Contains("some title") 
      ? workspacer.WindowStates.Floating 
      : null
)
```
or something similar.

**Comment by rickbutton** [comment]
Ah I see! Apologies, not enough coffee to read correctly.

"Untiled windows" are currently a hack and your suggested solution, to still pin "floating windows" to a "workspace" makes sense to me
